File: Java_MVP/app/src/main/java/com/gotoinc/requesin/view/home/HomeContract.java
package com.gotoinc.requesin.view.home;

import com.gotoinc.requesin.view._model.User;

import java.util.List;

/**
 * Created by Illia Derevianko on 26.11.18.
 * GoTo Inc.
 */
public interface HomeContract {
    void setUsers(List<User> users);
    void nextPage(boolean isExist, int nextPage);
}

File: Kotlin_MVP/app/src/main/java/com/gotoinc/requesin/view/home/HomeContract.kt
package com.gotoinc.requesin.view.home

import com.gotoinc.requesin.view.model.User

/**
 * Created by Illia Derevianko on 28.11.18.
 * GoTo Inc.
 */
interface HomeContract {
    fun setUsers(users: List<User>)
    fun nextPage(isExist: Boolean, nextPage: Int)
}